### ISSUE TITLE
Ability to configure Geoform with more layout flexibility

### ISSUE THREAD

**Issue by andrew-dc** [Esri/geoform-template-js#515]
It would be great to be able to format the layout of the Geoform with additional non-field elements in the Geoform builder. For example, the ability to add text/custom HTML blocks between fields, and to be able to group fields into categories with a heading above each. These sort of features exist in traditional forms but would be extremely useful for Geoforms as well.

**Comment by garciadb2187** [comment]
I would like to have the ability to move the map to the top of the form so users can see what information has been entered. This is important for my users since spatial relationships and time are a concern.

**Comment by driskull** [comment]
Yes, we'll add this option when we make the 4x JSAPI version of this app.

**Comment by zhujingx** [comment]
Yes. like to see could customize organize the text boxes. For instance, I have a long list of attribute fields info need to input, it will be much user friendly  if I can group the text boxes and give it a group title.
